package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "eslint-config-digitalbazaar",
+  "version": "1.0.0",
+  "description": "basic eslint config for digital bazaar projects",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/digitalbazaar/eslint-config-digitalbazaar.git"
+  },
+  "keywords": [
+    "eslint",
+    "config"
+  ],
+  "author": "liminal18",
+  "license": "ISC",
```

Comment:
I think this would be BSD-3 Licensed.  Add this license https://github.com/veres-one/did-veres-one/blob/master/LICENSE

Reply:
Except copyright 2019 by DIgital Bazaar, not 2018 by Veres One.

Reply:
@aljones15 here's a license with all the proper info: https://github.com/digitalbazaar/crypto-ld/blob/master/LICENSE

Reply:
I just changed the license to { "license" : "BSD-3-Clause" }